_i18n/es.yml

```diff
@@ -585,7 +585,7 @@ policies:
 
       ### Evaluación de Impacto de la Privacidad
       Vea nuestra [Evaluación del impacto
-      sobre la privacidad](site.baseurl/assets/docs/privacy-impact-assessment.pdf).
+      sobre la privacidad](site.baseurl/docs/privacy-impact-assessment.pdf).
```

Comment:
thanks for updating the URL here too! it's exactly what I was going to make sure we did